Lib/xmlrpc/client.py

```diff
@@ -245,41 +245,12 @@ def __repr__(self):
 
 ##
 # Backwards compatibility
-
 boolean = Boolean = bool
 
-##
-# Wrapper for XML-RPC DateTime values.  This converts a time value to
```

Comment:
I'm guessing that this comment is outdated, since it seems like `_iso8601_format` defined below does not accept strings or time tuples...